apisix/plugins/prometheus/exporter.lua

```diff
@@ -171,6 +171,8 @@ function _M.log(conf, ctx)
 
     metrics.bandwidth:inc(vars.bytes_sent,
         gen_arr("egress", route_id, service_id, consumer_name, balancer_ip))
+
+    core.log.info("prometheus run report")
```

Comment:
Can we check the result of prometheus instead of adding a new log just for testing?

Reply:
this is not just for testing purposes. Currently, there is no log output when prometheus reports data. I think adding this log will also help developers watch it while developing and debugging.

what do you think?

Reply:
A hit will increase the metric: https://github.com/apache/apisix/blob/1ac7166e80fdd502cc6a8a60ef58f415d3c0e56c/apisix/plugins/prometheus/exporter.lua#L153

There is no need to add another way.